packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -886,18 +972,22 @@ class OpacityLayer extends ContainerLayer {
   /// (as described at [Layer]).
   int alpha;
```

Comment:
this is mutable; would it make sense for RenderOpacity to just mutate this instead of creating a new layer each time?

Reply:
Yes, that makes sense to me. On the other hand, Layer is usually very cheap to construct so this probably won't make a difference. If it makes a difference, I'll probably make a change in a new PR since this is orthogonal to what we're doing here.